apollo-router/Cargo.toml

```diff
@@ -120,7 +120,8 @@ reqwest = { version = "0.11.11", default-features = false, features = [
     "json",
     "stream",
 ] }
-router-bridge = "0.1.3"
+# router-bridge = "0.1.3"
+router-bridge = { git = "https://github.com/apollographql/federation-rs.git", branch = "garypen/1519-slghtly-better-errors"}
```

Comment:
I presume this fix is going to wait for 0.1.4?

Reply:
yes